### PR TITLE
refactor static_website_tls: shortcuts for tls

### PR DESCRIPTION
**Pull request by azul**
taken from https://github.com/Drup/No.
thanks @drup

### CODE REVIEW

File: static_website_tls/dispatch.ml [hunk 1]
```diff
@@ -36,71 +35,53 @@ module Dispatch (C: CONSOLE) (FS: KV_RO) (S: HTTP) = struct
...
+  let start c http data keys () =
     tls_init keys >>= fun cfg ->
-    (* 31536000 seconds is roughly a year *)
-    let header = Cohttp.Header.init_with "Strict-Transport-Security" "max-age=31536000" in
-    let https flow = Dispatch_https.serve c flow (Dispatch_https.dispatcher ~header data) in
-    let http  flow = Dispatch_http.serve  c flow Dispatch_http.redirect in
-    S.listen_tcpv4 stack ~port:4433 (with_tls c cfg ~f:https);
-    S.listen_tcpv4 stack ~port:8080  http;
-    S.listen stack
+    let tcp = `TCP 4433 in
```
Comment (Drup) [inline static_website_tls/dispatch.ml]: Is this still supposed to be 4433 while the other is 443 ?

Comment (Drup) [inline static_website_tls/dispatch.ml]: (Maybe add a key for the two ports ? :p)

Comment (hannesm) [inline static_website_tls/dispatch.ml]: I prefer 4433, our examples should be runnable without root permissions

**Comment by azul** [comment]
@Drup thanks for catching that. fixed.

**Comment by samoht** [comment]
There is a conflict after merging #128. Can you rebase please? thx!

**Comment by azul** [comment]
:relaxed: @samoht rebased :recycle:

**Comment by samoht** [comment]
Thanks!

**Pull request merged**